build-rpi-raspbian/first-boot.sh

```diff
@@ -23,6 +23,10 @@ sed -i 's/\r$//' /etc/openhabian.conf
 source /etc/openhabian.conf
 echo "OK"
 
+echo -n "$(timestamp) [openHABian] Starting webserver... "
+sh /boot/webif.sh start
+echo "OK"
```

Comment:
Not necessarily...

Reply:
added a check if the web server is running with according output to the log